.github/scripts/findUnusedStyles.ts

```diff
@@ -0,0 +1,561 @@
+import * as fs from 'fs';
+import {globSync} from 'glob';
+import * as path from 'path';
+import * as ts from 'typescript';
+import TSCompilerUtils from '../../scripts/utils/TSCompilerUtils';
+
+type StyleDefinition = {
+    key: string;
+    file: string;
+    line: number;
+    column: number;
+};
+
+// Static patterns and constants - created once
+const STYLE_FILE_EXTENSIONS = '**/*.{ts,tsx,js,jsx}';
+const STYLE_KEY_SKIP_PATTERNS = ['default', 'exports', 'module', 'require', 'import', 'from', 'theme', 'colors', 'variables', 'CONST', 'Platform', 'StyleSheet'];
```

Comment:
This looks like it's only ever used with an `.includes`, so let's use a Set instead. I know it might not matter for performance in this context, but it sets a good example to use the appropriate data structure for quick lookups:

```suggestion
const STYLE_KEY_SKIP_PATTERNS = new Set(['default', 'exports', 'module', 'require', 'import', 'from', 'theme', 'colors', 'variables', 'CONST', 'Platform', 'StyleSheet']);
```